File: frontend/src/app/model/jwt.token.ts
export class JwtToken {
  token: string;
  expireTime: Date;

  public constructor(init?: Partial<JwtToken>) {
    Object.assign(this, init);

    if(init.expireTime){
      this.expireTime = new Date(init.expireTime);
    }
  }
}

File: backend/module-system/src/main/java/com/fy/system/jwt/JwtToken.java
package com.fy.system.jwt;

import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Date;

@Getter
@Setter
@Builder
public class JwtToken {

    private String token;

    private Date expireTime;
}
